### PR TITLE
TASK3: Implemented simple hello module with cli parameters

### PR DESCRIPTION
**Pull request by vyosypenko**

### CODE REVIEW

**Comment by ekovalyov** [comment]
Please add me (ekovalyov@gmail.com) as reviewer.

**Comment by ekovalyov** [comment]
Your PR should contains only commits with solution to only 1 task solution.

**Review by ekovalyov — APPROVED** [review]

File: 03_module/module_hello.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{
+    printk(KERN_INFO "Goodbye hello module...\n");
+    printk(KERN_INFO "x - y = %d\n", x - y);
+}
+
+module_init(hello_start);
+module_exit(hello_end);
+
+MODULE_LICENSE("MIT");
+MODULE_AUTHOR("Vasyl Yosypenko");
```
Comment (ekovalyov) [inline 03_module/module_hello.c:28]: Please check your module usign checpatch.pl

**Comment by ekovalyov** [comment]
Please remove commit with task 4 from this pull request.

File: 03_module/module_hello.c (hunk 1 above)
Comment (vyosypenko) [reply 03_module/module_hello.c]: Done

**Comment by vyosypenko** [comment]
@ekovalyov , please review updated changes.

**Review by ekovalyov — APPROVED** [review]
The task done.

**Comment by vyosypenko** [comment]
@ekovalyov, if everything is ok, please merge. 
Or can I merge if MR is approved?

**Pull request merged**